examples/istio/helidon-config/src/main/java/io/helidon/examples/config/GreetResource.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.examples.config;
```

Comment:
The package structure should reflect the module structure. In this case it should probably be `io.helidon.examples.istio.config`

Reply:
We don't have istio in package structure i.e. absolute path is helidon/examples/istio/helidon-config/src/main/java/**io/helidon/examples/config/GreetResource.java**